op-batcher/batcher/batch_submitter.go

```diff
@@ -73,7 +73,11 @@ func Main(version string, cliCtx *cli.Context) error {
 				l.Error("error starting metrics server", err)
 			}
 		}()
-		opmetrics.LaunchBalanceMetrics(ctx, l, registry, "", batchSubmitter.L1Client, batchSubmitter.From)
+		l1Client, err := dialEthClientWithTimeout(ctx, cfg.L1EthRpc)
+		if err != nil {
+			l.Error("Failed to connect to L1 provider to enable ")
```

Comment:
This feels like an unfinished sentence (at least the trailing space seems weird)

Reply:
HAHAHA you caught me 😅